docs/ai-developers/collaboration.md

```diff
@@ -0,0 +1,33 @@
+---
+# Page settings
+layout: default
+keywords:
+comments: false
+
+# Hero section
+title: AI Developers
+
+# extralink box
+extralink:
+    title: All Docs
+    title_url: '/docs'
+    external_url: false
+    description: Claims.
+
+# Market Place
+dev_news: true
+
+# Micro navigation
+micro_nav: true
+
+---
+
+# Collaboration
+Once an organization has been set up its possible to add members to it. This provides an easy way to collaborate with other developers and also distribute responsibilities.
+Members of an organization can do the following
+* Create services
+* Manage regions for a service
+* Edit / Delete Services
+
+Members **cannot perform any operation at an organization level**. 
+Organization level activities such as editing the organization etc can only be done by the owner
```

Comment:
let me recheck , this point , Members can Edit org data , but only the owner can publish

Reply:
Good pickup. Will rephrase it to publishing an organization